cli/replicaset/cmd/roles.go

```diff
@@ -79,37 +79,48 @@ func RolesAdd(ctx RolesAddCtx) error {
 	statusReplicasets(replicasets)
 	fmt.Println()
 
+	action := []string{"Add", "to"}
+	if replicaset.IsChangeRolesFuncEqual(changeRoleFunc, replicaset.RemoveRole) {
+		action = []string{"Remove", "from"}
+	}
```

Comment:
Comparing functions is usually a bad idea. At least, you should to do it via a switch with all cases.

Reply:
Refactored using polymorphism ideology.